docs/reference/scripting/painless.asciidoc

```diff
@@ -1,32 +1,34 @@
 [[modules-scripting-painless]]
 == Painless scripting language
 
-_Painless_ is a simple, secure scripting language designed specifically for use
```

Comment:
FYI, the audience for the reference part of the guide is highly technical users.